net-dns/yaddns/yaddns-1.1.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="A small dynamic dns client with multiple servers support and a high flexibility"
```

Comment:
Your `SUMMARY` is [too long][0]. Please use a shorter one.

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#summary